x-pack/test/security_solution_cypress/cypress/e2e/explore/hosts/misconfiguration_contextual_flyout.cy.ts

```diff
@@ -0,0 +1,232 @@
+/*
```

Comment:
Thanks for adding the test. Looks like it's interacting with alerts page and the flyout, shall we move it to x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/expandable_flyout?

Reply:
ah kk 
So I placed that test in the current folder because @machadoum suggested to put it here 
would it be better if i move it to x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/expandable_flyout instead ? @machadoum

Reply:
@animehart If the tested feature is only displayed inside the expandable flyout, placing the test where @angorayc  suggested might make sense. But that is up to the @elastic/security-threat-hunting-investigations team to decide since they own the `cypress/e2e/investigations` folder. I strongly suggest you create a folder for your team (cloud_security_posture?) so you don't depend on other teams for code reviews.

Reply:
@machadoum 
As discussed before, we were planning to add our own folder later, the reason being is because adding a specific folder for our team now will involve further steps to to make the new test to be ran on CI and because of this time constraint we think its better for us to put our test in existing folder for now